Database-Files/dummy.sql

```diff
@@ -176,9 +214,180 @@ insert into responsibilities (respDesc)
+insert into comp_Band(compID, bandID, compDesc)
+values (1, 9, 'You take a balanced view of wider impact on the organisation when making significant changes' );
+insert into comp_Band(compID, bandID, compDesc)
+values (2, 9, 'You role model company values, even when there is significant risk in doing so' );
+insert into comp_Band(compID, bandID, compDesc)
+values (3, 9, 'You understand that proposed innovation and changes should have a clear link to improving the business results delivered' );
+insert into comp_Band(compID, bandID, compDesc)
+values (4, 9, 'You manage expectations so customers always feels valued and have a positive experience of Kainos' );
+insert into comp_Band(compID, bandID, compDesc)
+values (5, 9, 'You tailor development approaches to suit the needs of each team member' );
+insert into comp_Band(compID, bandID, compDesc)
+values (6, 9, 'You organise people and resources to successfully achieve both short and medium term objectives' );
+insert into comp_Band(compID, bandID, compDesc)
+values (7, 9, 'You provide opportunities for others to learn functional and technical skills and concepts' );
+
+insert into band_Training(bandID, trainingID) values (1, 1);
+insert into band_Training(bandID, trainingID) values (1, 2);
+insert into band_Training(bandID, trainingID) values (1, 3);
+insert into band_Training(bandID, trainingID) values (1, 4);
+insert into band_Training(bandID, trainingID) values (5, 5);
+insert into band_Training(bandID, trainingID) values (6, 6);
+insert into band_Training(bandID, trainingID) values (7, 7);
+insert into band_Training(bandID, trainingID) values (8, 1);
+insert into band_Training(bandID, trainingID) values (9, 2);
+
+
+insert into resp_Band(respID, bandID) values (1, 1);
+insert into resp_Band(respID, bandID) values (2, 2);
+insert into resp_Band(respID, bandID) values (3, 3);
+insert into resp_Band(respID, bandID) values (4, 4);
+insert into resp_Band(respID, bandID) values (5, 5);
+insert into resp_Band(respID, bandID) values (6, 6);
+insert into resp_Band(respID, bandID) values (7, 7);
+insert into resp_Band(respID, bandID) values (8, 8);
+insert into resp_Band(respID, bandID) values (9, 9);
+
+
+insert into resp_Role(respID, roleID) values (1, 1);
+insert into resp_Role(respID, roleID) values (2, 2);
+insert into resp_Role(respID, roleID) values (3, 3);
+insert into resp_Role(respID, roleID) values (4, 4);
+insert into resp_Role(respID, roleID) values (5, 5);
+insert into resp_Role(respID, roleID) values (6, 6);
+insert into resp_Role(respID, roleID) values (7, 7);
+insert into resp_Role(respID, roleID) values (8, 8);
+insert into resp_Role(respID, roleID) values (9, 9);
+insert into resp_Role(respID, roleID) values (1, 10);
+insert into resp_Role(respID, roleID) values (2, 11);
+insert into resp_Role(respID, roleID) values (3, 12);
+insert into resp_Role(respID, roleID) values (4, 13);
+insert into resp_Role(respID, roleID) values (5, 14);
```

Comment:
Can we get a screenshot of this working

Reply:
![image](https://user-images.githubusercontent.com/30292929/68661078-46a5dc00-0532-11ea-8e05-2b3f49ab64d3.png)